content/applications/inventory_and_mrp/inventory/product_management/product_tracking/packaging.rst

```diff
@@ -140,4 +140,50 @@ used for the product. The chosen packaging is displayed on the :abbr:`SO (Sales
       :align: center
       :alt: Assign packagings on the Sales Order Line.
 
+.. _inventory/product_management/packaging-route:
+
+Routes for packaging
+====================
+
+When receiving packagings, by default, they follow the warehouse's :doc:`configured reception route
+<../../shipping_receiving/daily_operations/shipments_deliveries>`. To **optionally** set up a
+packaging-specific route, go to :menuselection:`Inventory app --> Configuration --> Routes`.
+
+.. important::
+   The :guilabel:`Product Packagings`, :guilabel:`Storage Locations`, and :guilabel:`Multi-Step
+   Routes` features (found by going to :menuselection:`Inventory app --> Configuration -->
+   Settings`) are required.
+
+.. seealso::
+   :doc:`../../warehouses_storage/inventory_management/use_routes`
+
+Create route
+------------
+
+On the :guilabel:`Routes` page, click :guilabel:`New` or select a route that is **not** for a
```

Comment:
```suggestion
On the :guilabel:`Routes` page, click :guilabel:`New`, or select a route that is **not** for a
```

Reply:
I can see in the caption of the figure below, the "Warehouses" *and* "Products" checkboxes are unticked. In this sentence, you mention that it must *not* be for a warehouse, but does it also need to be mentioned that it **must** have the "Products" checkbox unticked? Or is that not entirely necessary? Just curious -- something to consider during your revisions.